sdks/python/apache_beam/io/textio.py

```diff
@@ -927,7 +927,12 @@ def append(dest):
 
   @append_pandas_args(
       pandas.read_csv, exclude=['filepath_or_buffer', 'iterator'])
-  def ReadFromCsv(path: str, *, splittable: bool = True, **kwargs):
+  def ReadFromCsv(
+      path: str,
+      *,
+      splittable: bool = True,
+      label: str = 'ReadFromCsv',
```

Comment:
Would not this be backward incompatible if **kwargs already has a label key?

Reply:
yes, but I doubt users actually do that now.